pkg/user/signer.go

```diff
@@ -206,36 +206,35 @@ func (s *Signer) BroadcastTx(ctx context.Context, txBytes []byte) (*sdktypes.TxR
 	return resp.TxResponse, nil
 }
 
-// ConfirmTx periodically pings the provided node for the commitment of a transaction by its
-// hash. It will continually loop until the context is cancelled, the tx is found or an error
-// is encountered.
+// ConfirmTx waits for tx to be confirmed on the blockchain.
+// It stops waiting when the context is canceled.
```

Comment:
I think the previous godoc contains information that is still relevant. How about reverting the godoc change?

```go

```
```suggestion
// ConfirmTx periodically pings the provided node for the commitment of a transaction by its
// hash. It will continually loop until the context is cancelled, the tx is found or an error
// is encountered.
```

Reply:
yes, thank you

Reply:
Done